apps/src/netsim/NetSimRouterLogModal.html.ejs

```diff
@@ -29,7 +29,11 @@ function sortMarkupFor(sortKey) {
 <div class="modal-dialog modal-lg">
     <div class="modal-header">
       <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-      <h4 class="modal-title"><%= i18n.logBrowserHeader() %></h4>
+      <% if (isAllRouterMode) { %>
```

Comment:
Could you add a comment explaining the "parameters" (e.g. isAllRouterMode) to this view?